includes/polling/os/cyberpower.inc.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+$OIDs=array(
```

Comment:
Can you lower case all of your variables please, use _ to separate where you need to